tests/queries/0_stateless/02404_rocksdb_read_only.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
```

Comment:
```suggestion
# Tags: no-fasttest, no-parallel, no-ordinary-database
```

Check `01686_rocksdb.sql`

> -- Tag no-ordinary-database: Sometimes cannot lock file most likely due to concurrent or adjacent tests, but we don't care how it works in Ordinary database

Also, with `CLICKHOUSE_TMP` I don't see a reason to use `no-parallel`